src/argilla/server/apis/v1/handlers/datasets.py

```diff
@@ -99,6 +104,27 @@ def create_dataset(
     return datasets.create_dataset(db, dataset_create)
 
 
+@router.post("/datasets/{dataset_id}/annotations", status_code=status.HTTP_201_CREATED, response_model=Annotation)
+def create_dataset_annotation(
+    *,
+    db: Session = Depends(get_db),
+    dataset_id: UUID,
+    annotation_create: AnnotationCreate,
+    current_user: User = Security(auth.get_current_user),
+):
+    authorize(current_user, DatasetPolicyV1.create)
```

Comment:
The proper policy should be `create_annotation`, right?

Reply:
Yes, you're right. `create_annotation` will be present for dataset status checks.